ddl/backfilling.go

```diff
@@ -56,8 +60,65 @@ const (
 	typeUpdateColumnWorker     backfillWorkerType = 1
 	typeCleanUpIndexWorker     backfillWorkerType = 2
 	typeAddIndexMergeTmpWorker backfillWorkerType = 3
+
+	// InstanceLease is the instance lease.
+	InstanceLease = 60 // s
 )
 
+// enableDistReorg means whether to enable dist reorg. The default is enable.
+// TODO: control the behavior
+var enableDistReorg = atomicutil.NewBool(false)
+
+// DistReorgEnable enables dist reorg. It exports for testing.
+func DistReorgEnable() {
+	enableDistReorg.Store(true)
+}
+
+// DistReorgDisable disables dist reorg. It exports for testing.
+func DistReorgDisable() {
+	enableDistReorg.Store(false)
+}
+
+// IsDistReorgEnable indicates whether dist reorg enabled. It exports for testing.
+func IsDistReorgEnable() bool {
+	return enableDistReorg.Load()
+}
+
+// BackfillJob is for a tidb_ddl_backfill table's record.
+type BackfillJob struct {
+	ID            int64
+	JobID         int64
+	EleID         int64
+	EleKey        []byte
+	Tp            model.BackfillType
+	State         model.JobState
+	StoreID       int64
+	InstanceID    string
+	InstanceLease types.Time
+	Mate          *model.BackfillMeta
+}
+
+// AbbrStr returns the BackfillJob's info without the Mate info.
+func (bj *BackfillJob) AbbrStr() string {
```

Comment:
directly name ```String``` or ```Str```？

Reply:
Since this function only shows some ID related info, we don't use `String` here (and we're not sure if we need this function later).